data/sql/updates/pending_db_world/rev_1605128047204479800.sql

```diff
@@ -0,0 +1,154 @@
+    WHERE `quest_done_H` > 0 AND `quest_done_A` = `quest_done_H`
+;
+
+-- REORDER PRIMARY KEY
+CREATE TEMPORARY TABLE IF NOT EXISTS `temp` select * from `dungeon_access_requirements` ORDER BY dungeon_access_id, faction;
+TRUNCATE `dungeon_access_requirements`;
+ALTER TABLE `dungeon_access_requirements`; 
+INSERT INTO `dungeon_access_requirements` SELECT * FROM `temp`;
+
+-- Verify result with this
+-- SELECT `id`, `quest_failed_text` FROM `access_requirement` WHERE `quest_failed_text` IS NOT NULL;
+
+
+-- Remove columns + rename table
+ALTER TABLE `access_requirement`
+DROP `item`,
+DROP `item2`,
+DROP `quest_done_A`,
+DROP `quest_done_H`,
+DROP `completed_achievement`,
+DROP `quest_failed_text`,
+RENAME TO `dungeon_access_template`,
+COMMENT='Dungeon/raid access template and single requirements';
+
+-- Rename columns
+ALTER TABLE `dungeon_access_template`
+CHANGE `mapId` `map_id` mediumint unsigned NOT NULL COMMENT 'Map ID from instance_template' AFTER `id`,
+CHANGE `difficulty` `difficulty` tinyint unsigned NOT NULL DEFAULT 0 COMMENT '5 man: 0 = normal, 1 = heroic, 2 = epic (not implemented) | 10 man: 0 = normal, 2 = heroic | 25 man: 1 = normal, 3 = heroic' AFTER `map_id`,
+CHANGE `level_min` `min_level` tinyint unsigned NULL AFTER `difficulty`,
+CHANGE `level_max` `max_level` tinyint unsigned NULL AFTER `min_level`,
+CHANGE `item_level` `min_avg_item_level` smallint unsigned NULL COMMENT 'Min average ilvl required to enter' AFTER `max_level`,
+CHANGE `comment` `comment` varchar(255) COLLATE 'utf8_general_ci' NULL COMMENT 'Dungeon Name 5/10/25/40 man - Normal/Heroic' AFTER `min_avg_item_level`;
+
+-- Add KEY CONSTRAINTS
+ALTER TABLE `dungeon_access_template` ADD CONSTRAINT `FK_dungeon_access_template__instance_template` FOREIGN KEY (`map_id`) REFERENCES `instance_template` (`map`) ON DELETE CASCADE ON UPDATE CASCADE;
+ALTER TABLE `dungeon_access_requirements` ADD CONSTRAINT `FK_dungeon_access_requirements__dungeon_access_template` FOREIGN KEY (`dungeon_access_id`) REFERENCES `dungeon_access_template` (`id`) ON DELETE CASCADE ON UPDATE CASCADE;
+
+
+-- Add the acore_strings
+DELETE FROM `acore_string` WHERE `entry` IN (882,883,884,885,886,887,888);
+INSERT INTO `acore_string` (`entry`, `content_default`) VALUES 
+(882, 'To enter, you must complete the following quest(s):'),
+(883, 'To enter, you must complete the following achievement(s):'),
+(884, 'To enter, you must have the following item(s) in your inventory:'),
+(885, '- Hint:'),
+(886, 'You cannot enter. Access requirements not met.'),
+(887, 'To be able to enter, your equipment\'s average item level must be superior or equal to %u. Your current equipment\'s average ilevel is: %u.'),
+(888, 'You must be below level %u to enter.'),
+(889, 'To enter, the group leader must have complete the following quest(s):'),
+(890, 'To enter, the group leader must have complete the following achievement(s):'),
+(891, 'To enter, the group leader must have have the following item(s) in your inventory:');
```

Comment:
```suggestion
(889, 'To enter, the group leader must have completed the following quest(s):'),
(890, 'To enter, the group leader must have completed the following achievement(s):'),
(891, 'To enter, the group leader must have the following item(s) in his inventory:');
```

Reply:
Better to use "in the inventory"

Reply:
why? i disagree

Reply:
"their" works too.

Because in wow there are two genders, Male and Female

Reply:
yes, it also concerned me before, i thought the right word was "its" but in fact, it's common to use "his", i had verified on grammar sites

Reply:
You sure? I understand 'its' would be wrong but I think 'their' is right

Reply:
"the group leader must have the following item(s) in their inventory"

makes no sense lol

Reply:
It actually does.
Because the singular "they" as a pronoun can be used as "their"
> Singular they is the use in English of the pronoun they or its inflected or derivative forms, them, their, theirs, and themselves (or themself), as an epicene (gender-neutral) singular pronoun

https://en.wikipedia.org/wiki/Singular_they

Reply:
no it makes no sense, http://www.kentlaw.edu/academics/lrw/grinker/LwtaGender_Neutral_Language.htm

last comment for me

Reply:
"Their" Possessive form.
It is the "Raid Leaders inventory", their inventory.

> "The patient should be told at the outset how much they will be required to pay."

The patient - One patient, and they use "they"
Would have been plural if they used patients.

I do not make the English grammar rules :D

Reply:
no and you don't understand them either